### PR TITLE
v2: add support for listing all keys

### PR DESCRIPTION
**Pull request by oxtoacart**
Updates tailscale/corp#22783

### CODE REVIEW

**Comment by oxtoacart** [comment]
I wrote a test client (not checked in) to test this agains the live environment and it worked.

File: v2/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ func main() {
 	oauthScopes := []string{"all:write"}
 	tailnet := os.Getenv("TAILSCALE_TAILNET")
 
-	&tsclient.Client{
+	client := &tsclient.Client{
```
Comment (oxtoacart) [inline v2/README.md:55]: Drive-by README fix.

**Review by mpminardi — APPROVED** [review]
